.gotestfmt/package.gotpl

```diff
@@ -22,11 +22,11 @@ This template contains the format for an individual package.
         {{- range . -}}
             {{- if or (not $settings.HideSuccessfulTests) (ne .Result "PASS") -}}
                 {{- if eq .Result "PASS" -}}
-                    {{ "  \033" }}[0;32m✅
+                    {{ "  \033" }}[0;32m✅{{if $settings.ShowTestStatus}} ({{.Result}}){{end}}
```

Comment:
Same as with GitHub, it would be better to only create one bracket for readability.

Reply:
Addressed, please take another look.